### PR TITLE
Automating dependency install 

### PR DESCRIPTION
**Pull request by MBtech**
Here's a simple update to automate os-specific dependency installation and some other useful stuff like adding googliser directory to PATH and giving it an alias. 

Addressing Issue #44.
So something like this is all a user should need to do to get `googliser` as a command in their cli
`curl <shortened url to install script> && chmod +x install.sh && . install.sh`

However, there'd need to be an update to readme. I think this could be kept as an experimental automated setup until it has been more thoroughly tested.

### CODE REVIEW

**Comment by MBtech** [comment]
Plus it seems like on my Ubuntu 18.04.2 LTS, I did need to install Imagemagick. So I have added that too in the install script. 
I haven't updated the readme because I actually couldn't figure out the git.io url shortener to get a custom url to the install.sh script.

File: install.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ then
 elif [ "$OSTYPE" == "Linux" ];
 then
      LINUX_DIST=$(cat /etc/os-release | grep "^NAME=" | cut -d '=' -f2 | tr -d '"')
-     if [[ $LINUX_DIST =~ "Debian" ]]
+     if [[ $LINUX_DIST =~ "Debian" ]] || [[ $LINUX_DIST =~ "Ubuntu" ]]
```
Comment (teracow) [inline install.sh]: These can be combined as `if [[ $LINUX_DIST =~ "Debian" || $LINUX_DIST =~ "Ubuntu" ]]`

**Review by teracow — CHANGES_REQUESTED** [review]
There's a bit of swapping between single test `[` and double test `[[`. We're using BASH so double tests are OK. No need for any POSIX compatibility here. ;)

Also, your stdout captures use a mix of backticks and shell-braces. Can we make these all shell-braces please? e.g. `OSTYPE=$(uname)`

**Review by teracow — COMMENTED** [review]
Thinking out-loud, maybe we should include the functions in your `install.sh` within the main `googliser.sh` script? Then a new user could run it as `./googliser.sh --install` to get everything setup. This has the advantage of only requiring the user to download a single file that does everything. And no-need to offer multiple scripts in the repo.

Thoughts?

File: install.sh (hunk 1 above)
Comment (MBtech) [reply install.sh]: Yes that's perfectly fine

**Comment by MBtech** [comment]
That's a good idea I think. Having one file is better than multiple for usability. 
Though right now I am also creating directory to place the googliser script in and then adding that directory to the PATH. 

I had thought about moving it `googliser.sh` to `/usr/local/bin` but that might require sudo access. Though to be fair we are assuming that a user is able to install dependencies anyways. Thoughts on that?

**Comment by teracow** [comment]
If your installer.sh creates a directory to put googliser in, then it becomes simpler for the user to just run a `git clone` instead. ;)

Personally, with regard to moving `googliser.sh` into `/usr/local/bin`, I keep my scripts in one location (on a network drive), then create symlinks in `/usr/local/bin` that point back to that network location. Yes, `sudo` may still be required to create the symlink.

**Review by teracow — APPROVED** [review]
It's a beginning, something we can adjust as we go. :)

**Pull request merged**